src/main/java/productDependency/CreateOrderPage.java

```diff
@@ -0,0 +1,1481 @@
+
+		/**
+		 * Method to click on Products tab after successful login.
+		 * 
+		 * @throws IOException
+		 */
+		public void clickParentOrder() throws IOException {
+			
+			CreateOrderPage sp = new CreateOrderPage(driver);
+			log.info("Click on ParentOrder");
+			Assert.assertTrue(clickParentOrder.isDisplayed());
+			clickParentOrder.click();
+			JavaScriptExec.sleep();
+		}
+		
+		@FindBy(how = How.XPATH, using = "//*[@id='edit_change_header_-3']/td[1]")
+		//*[@id="edit_change_header_-3"]/td[1]
+		
+		private WebElement clickParentOrder1;
+
+		/**
+		 * Method to click on Products tab after successful login.
+		 * 
+		 * @throws IOException
+		 */
+		public void clickParentOrder1() throws IOException {
+			
+			CreateOrderPage sp = new CreateOrderPage(driver);
+			log.info("Click on ParentOrder");
+			Assert.assertTrue(clickParentOrder1.isDisplayed());
+			clickParentOrder1.click();
+			JavaScriptExec.sleep();
+		}
+		
+		@FindBy(how = How.XPATH, using = "//*[@id='edit_change_header_-3']/td[1]")
+		//*[@id="edit_change_header_-3"]/td[1]
+	
+		
+		private WebElement clickParentOrder2;
+
+		/**
+		 * Method to click on Products tab after successful login.
+		 * 
+		 * @throws IOException
+		 */
+		public void clickParentOrder2() throws IOException {
+			
+			CreateOrderPage sp = new CreateOrderPage(driver);
+			log.info("Click on ParentOrder");
+			Assert.assertTrue(clickParentOrder2.isDisplayed());
+			clickParentOrder2.click();
+			JavaScriptExec.sleep();
+		}
+	
+	
+	
+	
+	// @FindBy(how=How.XPATH, using="//*[@id="dependencies-products-change_-3"]/tbody/tr[3]/td[1]/a/strong
+	private WebElement clickDependentProduct;
+
+		/**
+		 * Method to click on dependent product.
+		 * 
+		 * @throws IOException
+		 */
+	public void clickDependentProduct() throws IOException {
+		
+			CreateOrderPage sp = new CreateOrderPage(driver);
+			log.info("click on dependenct product.");
+			String DependentProduct = BasePage.getCellData(xlsxName, sheetName, 7, 0);
+			driver.findElement(By.xpath("//a[not(@href)]/strong[text()='" + DependentProduct + "']")).click();
+		}
+		
+	// @FindBy(how=How.XPATH, using="//*[@id="dependencies-products-change_-3"]/tbody/tr[3]/td[1]/a/strong
+	private WebElement clickDependentProduct1;
+
+		 		/**
+		 		 * Method to click on dependent product.
+		 		 * 
+		 		 * @throws IOException
+		 		 */
+	public void clickDependentProduct1() throws IOException {
+		 	CreateOrderPage sp = new CreateOrderPage(driver);
+		 	log.info("click on dependenct product.");
+		 	String DependentProduct = BasePage.getCellData(xlsxName, sheetName, 7, 2);
+		 	driver.findElement(By.xpath("//a[not(@href)]/strong[text()='" + DependentProduct + "']")).click();
+		 		}
+		 		
+	// @FindBy(how=How.XPATH, using="//*[@id="dependencies-products-change_-3"]/tbody/tr[3]/td[1]/a/strong
+	private WebElement clickDependentProduct2;
+
+		 		/**
+		 		 * Method to click on dependent product.
+		 		 * 
+		 		 * @throws IOException
+		 		 */
```

Comment:
@mayankchhajer Please remove the unused WebElements.

Reply:
Done